osu.Game/Overlays/Mods/ModCustomisationHeader.cs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+using osu.Game.Localisation;
+
+namespace osu.Game.Overlays.Mods
+{
+    public partial class ModCustomisationHeader : OsuHoverContainer
+    {
+        public override bool HandlePositionalInput => true;
```

Comment:
Is this needed? The inherited class `OsuHoverContainer` overrides `On-` input methods so it should already be true.

Reply:
Can't remember why I added that there, should be fine to remove.